src/propagate-job.ts

```diff
@@ -44,30 +71,6 @@ export async function run(): Promise<void> {
       return;
     }
 
```

Comment:
i'd include some sort of check here to avoid performing all the extra requests and computations if the transaction would fail. an example would be perhaps fetching `aggregateRoot` and `lastPropagatedRoot` and ensuring they are different. 

another thing that could be done is, once we get all the parameters for the transaction--so before sending the bundle--we could perform a `callStatic` and returning if an error is encountered to avoid sending the bundle to flashbots.